### DISCUSSION TITLE
如何对群里面发送一个图片

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#79] [Q&A] [answered]
### 涉及的编程语言

Java

### 疑问描述

请问一大佬 怎么对群里面发送一个图片呢

**Comment by yujumpjump** [comment]
不好意思大佬 我没有看到文档 现在看到了 打扰大佬了 哈哈哈哈

**Comment by yujumpjump** [comment, marked as answer]
大佬 你那个教程是监听朋友的信息 但是我想实现群里面监听信息然后发送一个图片 过去 怎么实现呢 我看MiraiGroupMessageEvent 这个接口没有相关的方法

**Reply by ForteScarlet** [reply]
```java
Group group = groupEvent.getGroup();
group.send(...); // 发送消息
```

**Reply by yujumpjump** [reply]
if(event.getMessageContent().getPlainText().contains("注册")){
            MiraiGroup group = event.getGroup();
            PathResource resource = Resource.of(Paths.get("image.png"));
            ResourceImage resourceImage = Image.of(resource);
            group.sendBlocking(resourceImage);
        }else {
            event.getGroup().sendBlocking("pb="+event.getAuthor().getNickname());
        } 
这是文我的源代码  但是他会出现这个错误  java.nio.file.NoSuchFileException: images.jpg

**Reply by yujumpjump** [reply]
大佬 可以写一段代码给我看看嘛 我很需要 麻烦大佬了

**Reply by ForteScarlet** [reply]
> java.nio.file.NoSuchFileException: images.jpg

错误很明显，你没有叫做 `images.png` 
的文件，自行检查自己的项目里是否存在指定文件

**Reply by yujumpjump** [reply]
好的 大佬 我已经解决了 有什么办法图片和文字一起发送吗

**Reply by ForteScarlet** [reply]
https://simbot.forte.love/docs/basic/messages#%E6%9E%84%E5%BB%BA%E6%B6%88%E6%81%AF%E9%93%BE

**Reply by yujumpjump** [reply]
谢谢大佬 已经解决

**Reply by yujumpjump** [reply]
大佬 我还有一个 问题想请教一下 就是今天在获取群公告的时候 他获取的竟然是一个空的 公告集合  请问这是什么情况呢

**Reply by yujumpjump** [reply]
会出现这个 错误 ： love.forte.simbot.event.EventListenerProcessingException: kotlinx.serialization.MissingFieldException: Field 'msg' is required for type with serial name 'net.mamoe.mirai.internal.contact.announcement.GroupAnnouncement', but it was missing at path: $

**Reply by ForteScarlet** [reply]
https://github.com/simple-robot/simpler-robot/issues/602

**Reply by yujumpjump** [reply]
大概 什么时候 可以修复好了 大佬

**Reply by ForteScarlet** [reply]
这是mirai的bug，问我没用

**Reply by yujumpjump** [reply]
好的 谢谢
